src/test/java/uk/gov/hmcts/reform/orgrolemapping/domain/service/DroolCivilStaffOrgRolesTest.java

```diff
@@ -1,234 +1,168 @@
+            } else {
+                assertNull(r.getAttributes().get(REGION_ID));
             }
-        });
+        }
 
-        roleAssignments.stream().filter(c -> c.getGrantType().equals(GrantType.STANDARD)).collect(Collectors.toList())
-                .forEach(r -> {
-                    assertEquals("CIVIL", r.getAttributes().get("jurisdiction").asText());
-                    if ("senior-tribunal-caseworker".equals(r.getRoleName())
-                        || "task-supervisor".equals(r.getRoleName())) {
-                        assertEquals("decision_making_work,access_requests",
-                                r.getAttributes().get("workTypes").asText());
-                    }
-                    assertEquals("region1", r.getAttributes().get("region").asText());
-                    assertEquals(cap.getPrimaryLocationId(), r.getAttributes().get("primaryLocation").asText());
-                });
-    }
 
+        String expectedWorkTypes;
+        if (roleId.equals("1") && r.getRoleName().equals("task-supervisor")) {
+            expectedWorkTypes = "decision_making_work,access_requests";
+        } else {
+            expectedWorkTypes = expectedRoleNameWorkTypesMap.get(r.getRoleName());
+        }
 
-    static Stream<Arguments> generateData() {
-        return Stream.of(
-                Arguments.of("3", Arrays.asList("hmcts-admin", "hearing-centre-team-leader"),
-                        2, Collections.singletonList("hearing_work,access_requests")),
-                Arguments.of("10", Arrays.asList("hmcts-ctsc", "ctsc"),
-                        2, Arrays.asList("routine_work")),
-                Arguments.of("9", Arrays.asList("hmcts-ctsc", "ctsc-team-leader"),
-                        2, Arrays.asList("routine_work,access_requests")),
-                Arguments.of("4", Arrays.asList("hmcts-admin", "hearing-centre-admin"),
-                        2, Collections.singletonList("hearing_work")),
-                Arguments.of("1", Arrays.asList("senior-tribunal-caseworker","hmcts-legal-operations"),
-                        2, List.of("decision_making_work,access_requests")),
-                Arguments.of("2", Arrays.asList("tribunal-caseworker","hmcts-legal-operations"),
-                        2, List.of("decision_making_work")),
-                Arguments.of("6", Arrays.asList("hmcts-admin", "nbc-team-leader"),
-                        2, Arrays.asList("routine_work,access_requests")),
-                Arguments.of("11", Arrays.asList("hmcts-admin","national-business-centre"),
-                        2, Arrays.asList("routine_work"))
-        );
+        String actualWorkTypes = null;
+        if (r.getAttributes().get("workTypes") != null) {
+            actualWorkTypes = r.getAttributes().get("workTypes").asText();
+        }
+        assertEquals(expectedWorkTypes, actualWorkTypes);
     }
 
-    static Stream<Arguments> generateDatav11() {
+    static Stream<Arguments> generateDatav14() {
```

Comment:
Can you add a coment before the stream just listing the names of the arguments in this data, i.e. just a CSV list of the param names will do.

Reply:
Is it worth us reordering these so they match the order of confluence table?

Reply:
Reorder and comment added

Reply:
Resolved in 469ccf44cf3808543fe7dd380bba2d6cb9f5a6ce.